packages/gatsby-theme-newrelic/package.json

```diff
@@ -7,9 +7,9 @@
     "access": "public"
   },
   "scripts": {
-    "build": "gatsby build",
+    "build": "GATSBY_NEWRELIC_ENV=production gatsby build",
```

Comment:
why production vs staging?

Reply:
This is so it reports to the Staging account in NR1 with the rest of the data

Reply:
but to double double check, that is what we want? like if we run this locally, we want that data mixing into the same environment as our prod data. is that 👍 

if so, feel free to resolve this.